lib/system-checker.coffee

```diff
@@ -74,8 +74,7 @@ class SystemChecker
 
     # Try the packaged library inside the node_modules. `getDictionaryPath` is
```

Comment:
This comment doesn't look correct anymore now that it's been changed to use `spellchecker.getDictionaryPath()` :smile:.